src/services/users.service.js

```diff
@@ -0,0 +1,47 @@
+import { User } from '../models/user.model.js';
+
+async function consumeActivationToken(user) {
+  user.activationToken = null;
+  await user.save();
+}
+
+async function getAll() {
+  return User.findAll({
+    attributes: ['id', 'name', 'email', 'activationToken'],
+  });
+}
+
+async function getById(id) {
+  return User.findByPk(id);
+}
+
+async function getByEmail(email) {
+  return User.findOne({ where: { email } });
+}
+
+async function getByActivationToken(activationToken) {
+  return User.findOne({
+    where: { activationToken },
+    attributes: ['id', 'name', 'email'],
```

Comment:
The 'getByActivationToken' function excludes 'activationToken' from the returned attributes. If the token is needed for further processing, ensure it's handled securely.